README.rst

```diff
@@ -107,6 +107,7 @@ The following modules have been completed using a common API:
   * `CADC <https://astroquery.readthedocs.io/en/latest/cadc/cadc.html>`_: Canadian Astronomy Data Centre.
   * `ESASky <http://astroquery.readthedocs.io/en/latest/esasky/esasky.html>`_: ESASky is a science driven discovery portal providing easy visualizations and full access to the entire sky as observed with ESA Space astronomy missions.
   * `ESO Archive <http://astroquery.readthedocs.io/en/latest/eso/eso.html>`_
+  * `FIRST <http://astroquery.readthedocs.io/en/latest/first/first.html>`_: Faint Images of the Radio Sky at Twenty-cm. 20-cm radio images of the extragalactic sky from the VLA.
```

Comment:
This link won't exist without adding the narrative docs

Reply:
Oh thanks, I missed that part.  I assume we need documentation for both the image_cutouts and image_cutouts.first modules.  I will model this after the solarsystem module, which has some nested modules.

A question (for @keflavich too): I see that the solarsystem module has entries at the main level too.  E.g. astroquery.jplhorizons is identical to astroquery.solarsystem.jpl.horizons.  The solarsystem.jpl.horizons `__init__.py` file simply imports everything from the top level astroquery.jplhorizons module.  Should I be doing that for first as well?

Reply:
No, just do the `astroquery.image_cutouts.first`

Reply:
OK, docs are done and passed the tests.